ground/uavobjgenerator/uavobjgenerator.pro

```diff
@@ -5,14 +5,14 @@ QT += xml
 QT -= gui
 
 macx {
-    QMAKE_CFLAGS_X86_64 += -mmacosx-version-min=10.7
-    QMAKE_CXXFLAGS_X86_64 = $$QMAKE_CFLAGS_X86_64
+    QMAKE_MACOSX_DEPLOYMENT_TARGET=10.9
```

Comment:
I appreciate this change, but is it perhaps out of scope? The commit message for 6a703c9a91598e99165f3a75a8ee3e286c7bc28d doesn't describe why this change is necessary or what it's intended to do.